tox.ini

```diff
@@ -25,60 +25,29 @@ passenv =
 [testenv:fmt]
 description = Apply coding style standards to code
 deps =
-    black
-    ruff
+    -r{toxinidir}/fmt-requirements.txt
 commands =
+    isort {[vars]all_path}
     black {[vars]all_path}
-    ruff --fix {[vars]all_path}
 
 [testenv:lint]
 description = Check code against coding style standards
 deps =
-    black
-    ruff
```

Comment:
This is taken from the [template-operator](https://github.com/canonical/template-operator/blob/main/tox.ini), I am not sure if we want to keep it or not. On one hand it makes sense to be using the same tools on all our charms (we would probably have to update the `tox.ini`s in our other charms as well), on the other hand we want to stay up to date with what other people are using.

@massigori @natalian98 what do you think? Should we use isort for now and update our dependencies across all our charms in the future or should we use the latest tools in our new charms and update the existing ones separately?

Reply:
I'd say to keep using isort as in other charms and then update it to ruff collectively, once we have a solid process for upgrading dependencies. That's because I'm unsure if ruff has the same features as isort and we want to keep all charms standardized in terms of linting. @gruyaume what would you advise?

Reply:
currently, linting test uses black, flake8, and isort. Let me know if I should include anything else.